### PR TITLE
Equinix Metal: Update Image to Ubuntu 20.04

### LINKED ISSUE #95: Update Ubuntu images to 18.04
## Expected Behaviour

Ubuntu 16.04 is now EOL, and only paid "ESM" support / updates are available.

Update all provisioners to use Ubuntu 18.04 images and then test each.

Otherwise, images may be removed without warning, such as in: https://github.com/inlets/inlets-operator/issues/123, resulting in users being unable to use inletsctl/inlets-operator.

### PR DESCRIPTION
**Pull request by dirien**
Signed-off-by: Engin Diri <engin.diri@mail.schwarz>

Fixes #95 

## How Has This Been Tested?

```bash
 go run . create -p equinix-metal --project-id=yyy --access-token=xxx --region=fr2
Using provider: equinix-metal
Requesting host: jolly-bassi0 in fr2, from equinix-metal
Host: 76a80375-b701-4deb-ad01-e823eace1efd, status: 
[1/500] Host: 76a80375-b701-4deb-ad01-e823eace1efd, status: provisioning
[2/500] Host: 76a80375-b701-4deb-ad01-e823eace1efd, status: provisioning
[3/500] Host: 76a80375-b701-4deb-ad01-e823eace1efd, status: provisioning
[4/500] Host: 76a80375-b701-4deb-ad01-e823eace1efd, status: provisioning
...
[36/500] Host: 76a80375-b701-4deb-ad01-e823eace1efd, status: provisioning
[37/500] Host: 76a80375-b701-4deb-ad01-e823eace1efd, status: provisioning
[38/500] Host: 76a80375-b701-4deb-ad01-e823eace1efd, status: active
inlets PRO TCP (0.8.6) server summary:
  IP: 145.40.93.57
  Auth-token: xxx
```

Start Exit Node
```bash
inlets-pro tcp client --url "wss://145.40.93.57:8123" \
  --token "xxxx" \
  --upstream $UPSTREAM \
  --ports $PORTS

2021/07/27 06:53:05 Starting TCP client. Version 0.8.5 - 8db64ded51b1455cf4ad027c52bc9cfbeb55c4b3
2021/07/27 06:53:05 Licensed to: Engin Diri <Engin.Diri@mail.schwarz>, expires: 26 day(s)
2021/07/27 06:53:05 Upstream server: localhost, for ports: 8080
inlets-pro client. Copyright Alex Ellis, OpenFaaS Ltd 2020
INFO[2021/07/27 06:53:06] Connecting to proxy                           url="wss://145.40.93.57:8123/connect"
INFO[2021/07/27 06:53:07] Connection established.. OK.
```
Test Local Service:

```bash
➜  ~ curl 145.40.93.57:8080
<!DOCTYPE html>
<html>
<head>
<title>Welcome to nginx!</title>
<style>
    body {
        width: 35em;
        margin: 0 auto;
        font-family: Tahoma, Verdana, Arial, sans-serif;
    }
</style>
</head>
<body>
<h1>Welcome to nginx!</h1>
<p>If you see this page, the nginx web server is successfully installed and
working. Further configuration is required.</p>

<p>For online documentation and support please refer to
<a href="http://nginx.org/">nginx.org</a>.<br/>
Commercial support is available at
<a href="http://nginx.com/">nginx.com</a>.</p>

<p><em>Thank you for using nginx.</em></p>
</body>
</html>
```

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [x] read the [CONTRIBUTION](https://github.com/inlets/inlets/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

File: cmd/create.go [hunk 1]
```diff
@@ -425,8 +425,8 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 	} else if provider == EquinixMetalProvider {
 		return &provision.BasicHost{
 			Name:     name,
-			OS:       "ubuntu_16_04",
-			Plan:     "t1.small.x86",
+			OS:       "ubuntu_20_04",
+			Plan:     "c3.small.x86",
```
Comment (alexellis) [inline cmd/create.go:429]: Can you take this as a flag? `--plan`

Use the default, unless someone has given an override. You'll see how I did it for the operator.

Comment (dirien) [reply cmd/create.go:429]: Is this not already in place with -> https://github.com/inlets/inletsctl/blob/ed0bcc767eb9357bef75062dee746684b97eb0bf/cmd/create.go#L277

The default is the on writen, and after the hostReq get checked for planoverwrite?

Comment (cpanato) [reply cmd/create.go:429]: good call the t1 instances will be deprecated

Comment (alexellis) [reply cmd/create.go:429]: OK we are doing that in a weird non-obvious way, I'd much prefer to determine the plan in this function than to mutate the result of the function from a command line flag after the fact.

That's why I missed this. I'll merge this, but would you like to work on the proposed behaviour change? That's how the operator works ATM:

https://github.com/inlets/inlets-operator/blob/master/controller.go#L553-L569

Comment (alexellis) [reply cmd/create.go:429]: @dirien could you refactor how we handle the plan as per the operator?

Comment (dirien) [reply cmd/create.go:429]: Created a issue for it #111

**Pull request merged**